react-ui-kit/src/Checkbox/CheckboxComponent.ts

```diff
@@ -0,0 +1,46 @@
+import styled, { css } from 'styled-components';
+
+import { ITheme, DefaultTheme } from '../palette/variables';
```

Comment:
Мне кажется, что так чуть правильнее. 1 - испортируем тип как тип, 2 - не идём в недра palette, а берём то, что он отдаёт во внешний мир
```suggestion
import { DefaultTheme } from '../palette';
import type { ITheme } from '../palette';
```

Reply:
1 - я бы не стал выделять тип, кажется лишнее, в вот в потроха лезть да, надо бы поправить

Reply:
Ну не знаю. Как по мне, тип логичнее импортировать как тип

Reply:
Ок, поменяю везде тогда